src/coreclr/src/vm/interopconverter.cpp

```diff
@@ -194,6 +202,12 @@ IUnknown *GetComIPFromObjectRef(OBJECTREF *poref, ComIpType ReqIpType, ComIpType
         if (pFetchedIpType != NULL)
             *pFetchedIpType = ReqIpType;
 
+        if (pvObj != NULL)
```

Comment:
Let's fold this into a new `else` block in the `if`/`else if` chain above. That way the `pvObj` can always represent the return.